src/editor.js

```diff
@@ -104,6 +115,58 @@ CKEDITOR.define( [
 					};
 				}
 			}
+
+			function prepareData() {
+				// At this stage we have the editor initial data at its "pure" form. There are two possibilities:
+				//
+				//   1. editor.setData() has been called before editor.init() - editor._data has been set. In this case
+				//      we don't want to change it because we assume that this is the data the end-developer wants to
+				//      have, no matter what.
+				//
+				//   2. An editable is available. In this case we take the initial data from editable.element. This must
+				//      be done at this stage because both fireCreator() and initEditables() will drive the data to
+				//      editable.view instead.
+				if ( typeof that._data != 'string' ) {
+					that._data = that.getData();
+				}
+			}
+
+			function fireCreator() {
+				// Take the name of the creator to use (config or any of the registered ones).
+				var creatorName = config.creator || Object.keys( that._creators )[ 0 ];
+
+				if ( creatorName ) {
+					// Take the registered class for the given creator name.
+					var Creator = that._creators[ creatorName ];
+
+					if ( !Creator ) {
+						throw( new Error( 'The "' + creatorName + '" creator was not found. Check `config.creator`.' ) );
+					}
+
+					// Create an instance of the creator.
+					that._creator = new Creator( that );
+
+					// Finally fire the creator. It may be asynchronous, returning a promise.
+					return that._creator.create();
+				}
+
+				return 0;
```

Comment:
My question was: what should happen if there is no creator? There should be a note that doing nothing (no error) is designed behavior, because it was not clear for me.